app/forms/providers/proceeding_merits_task/linked_children_form.rb

```diff
@@ -16,21 +16,25 @@ def value_list
       def save
         return false unless valid?
 
-        ActiveRecord::Base.transaction do
-          update_involved_children_on_proceeding
-        end
-        true
-      rescue StandardError
-        false
+        update_involved_children_on_proceeding
+      end
+
+      def save_as_draft
+        @draft = true
+        save! unless update_involved_children_on_proceeding || all_entries_blank?
```

Comment:
is this expected to call the save defined above? As without `alias :save!, :save` it won't

Reply:
was looking at this commit in isolation, i see the alias_mehod was added in a subsequent commit. 👍